ophyd/signal.py

```diff
@@ -189,6 +193,13 @@ def describe_configuration(self):
         return {}
 
 
+class PySignal(Signal):
```

Comment:
If it can not be used with `Device` what is the use of keeping `Signal` around as a public class at all?